client/src/Q&As/Answers.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const Answers = ({ questions, product }) => {
+  const [answers, setAnswer] = useState([]);
+
+  useEffect(() => {
+    if (questions) {
+      axios.get(`/qa/questions/${questions}/answers`)
+        .then((response) => {
+          console.log(response.data.results.photos);
+          return setAnswer(response.data.results);
+        })
+        .catch(() => {
+          console.log('cant get request from question API');
+        });
+    }
+  }, [questions]);
+
+  //load up to 2 answers on the page
+  //need to figure out how to hide rest of answers and show them when I click 'load more answers'
```

Comment:
I had to do the same thing. Let me know if you want some tips!

Reply:
oh awesome I'll let you know if I need some hints